api/accounts/accounts_test.go

```diff
@@ -108,19 +108,24 @@ var ts *httptest.Server
 func TestAccount(t *testing.T) {
 	initAccountServer(t)
 	defer ts.Close()
-	getAccount(t)
-	getAccountWithNonExisitingRevision(t)
-	getAccountWithGenesisRevision(t)
-	getAccountWithFinalizedRevision(t)
-	getCode(t)
-	getCodeWithNonExisitingRevision(t)
-	getStorage(t)
-	getStorageWithNonExisitingRevision(t)
-	deployContractWithCall(t)
-	callContract(t)
-	callContractWithNonExisitingRevision(t)
-	batchCall(t)
-	batchCallWithNonExisitingRevision(t)
+
+	for name, tt := range map[string]func(*testing.T){
```

Comment:
map will disturb the order, would you mind making it a struct array?

Reply:
@paologalligit Could you pitch in to make the decision call please ?

Myself and @libotony have been chatting offline about the array vs maps and there are good points on each side, and we may have come to decision paralysis hehe. 

What would be your pref ?

Reply:
I'd say the map approach is fine, we don't care about the order of the tests, we can have them run in any order